sensor_header_msg.cpp

```diff
@@ -1,5 +1,4 @@
 #include "sensor_header_msg.h"
-#include "FreeRTOS.h"
```

Comment:
👏 Nice — just a copy-pasta error or did we have an malloc call in here previously?

Reply:
I can't remember 😛 I suspect maybe a malloc though? I vaguely remember mallocing in this repo.